rr_light/reflection_refraction.py

```diff
@@ -141,12 +143,10 @@ def matplotlib_graph(p1: float, p2: float, p3: float, n1: float, n2: float) -> s
     plt.gca().axes.get_yaxis().set_ticks([])
     
     # Incident beam
-    dc1 = [p1[0], p1[1]]
-    plt.arrow(x=DISTANCE*p1[0], y=DISTANCE*p1[1], dx=DISTANCE*-dc1[0], dy=DISTANCE*-dc1[1], facecolor=COLOR_OF_INCIDENT_BEAM, width=0.4, head_width=1, head_length=1.5, length_includes_head=True)
+    plt.arrow(x=DISTANCE*p1[0], y=DISTANCE*p1[1], dx=DISTANCE*-p1[0], dy=DISTANCE*-p1[1], facecolor=COLOR_OF_INCIDENT_BEAM, width=0.4, head_width=1, head_length=1.5, length_includes_head=True)
```

Comment:
Why I have a linter error here? It's not a var

Reply:
You have typed `p1: float` and the other points wrongly in the methods definition:

`def matplotlib_graph(p1: float, p2: float, p3: float, n1: float, n2: float)` 

and it should be

`def matplotlib_graph(p1: List[float], p2: List[float], p3: List[float], n1: float, n2: float)`